src/scripts/preboot-sim.sh

```diff
@@ -1,12 +1,27 @@
 #!/usr/bin/env bash
 
+UDID_not_found() {
```

Comment:
Hi @bdellegrazie sorry for the late follow up on this and thank you for the contribution!

My main comment on this is I am not sure it entirely solves the issue here. The error message here returns two separate lists, the sim versions and the device names. This still would not provide a list of correct combinations for simulators installed in the image.

We have recently changed the software manifests to include a proper list of sim version + device combination along with the uuid of the simulator (example: https://circle-macos-docs.s3.amazonaws.com/image-manifest/v10821/manifest.txt)

I feel this kind of list being returned might be more beneficial, so something like:

```$(xcrun simctl list devices | tail -n +2)``` for the simulators

```$(xcrun simctl list pairs | tail -n +2)``` for the simulator pairs

Reply:
Sure, will do - thanks for the information.

Reply:
@BytesGuy 
I have rebased on circleci/main and updated in accordance with your suggestion for the error message.
Please be aware that I'm doing this blind - I don't have a mac to test this on.

Reply:
@bdellegrazie Ahh ok, let me test this locally for you and see if there are any other suggestions :)